core/unix/signal.c

```diff
@@ -6523,9 +6521,15 @@ execute_handler_from_dispatch(dcontext_t *dcontext, int sig)
     return true;
 }
 
-/* Sends a signal to a currently-native thread.  dcontext can be NULL. */
+/* Sends a signal to a currently-native thread.
+ * dcontext can be NULL.
+ * cur_xsp is the stack pointer seen by the DR signal handler. May be NULL for
```

Comment:
"seen" is confusing: it sounds like the one in the frame, the interrupted sp, as that's what the handler "sees".  Please reword s/seen/initially executed on/ or something similar.

Reply:
Done